cypress/e2e/read-receipts/high-level.spec.ts

```diff
@@ -906,8 +899,20 @@ describe("Read receipts", () => {
                 saveAndReload();
                 assertUnread(room2, 3);
             });
-            // XXX: fails because on CI the count is 2 instead of 3. Must be a timing issue.
-            it.skip("A room where all threaded edits are read is still read after restart", () => {
+            it("A room where all threaded edits are read is still read after restart", () => {
+                goTo(room2);
+                receiveMessages(room2, ["Msg1", threadedOff("Msg1", "Resp1"), editOf("Resp1", "Edit1")]);
+                assertUnread(room2, 2);
+                openThread("Msg1");
+                assertRead(room2);
+                goTo(room1);
```

Comment:
Why does to test go to room1 here? Is it because of that is should view room 1 after reload? Can you add a small comment here?

Reply:
Yeah I will add a comment, thanks. Generally these tests go back to room1 to avoid immediately reading any messages that appear later. Here we want to ensure that we don't accidentally suppress any unreadness on reload by already being in the room.

Reply:
Done in f208b70cbf